arena.js

```diff
@@ -0,0 +1,37 @@
+import Gladiator from './gladiator.js';
+
+export default class Arena {
+    constructor(name, gladiators) {
+        this._name = name;
+        this.gladiators = [];
+
+    }
+
+    get name() {
+        return this._name.charAt(0).toUpperCase() + this._name.slice(1);
+    }
+    addGladiator(name) {
+        if (this.gladiators.length < 2)
+            this.gladiators.push(name);
+    }
+
+    fight() {
+        if (this.gladiators.length <= 2) {
+            if (this.gladiators[1].weapon === this.gladiators[0].weapon) {
+                this.gladiators.pop();
```

Comment:
.pop() here removes the last element of the the gladiators array. 

one way to remove an element is to use a method called .splice()
here is a link on how to use it
[splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice)